examples/buzzer/main.cc

```diff
@@ -30,7 +30,7 @@ static bsp::BuzzerNoteDelayed Mario[] = {
     {Note::Silent, 240}, {Note::Mi3M, 80},   {Note::Silent, 240},
     {Note::Do1M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
     {Note::Silent, 240}, {Note::So5M, 80},   {Note::Silent, 560},
-    {Note::So5L, 80},
+    {Note::So5L, 80},    {Note::Silent, 0},  {Note::Finish, 0}
```

Comment:
The test on board shows that Finish is necessary or the next song will not play.

Reply:
Let's put a separate fix on those things as they are unrelated to this pull request.